.changeset/fresh-tables-clap.md

```diff
@@ -0,0 +1,77 @@
+---
+"@meilisearch/instant-meilisearch": minor
+---
+
+- Change the behavior of the `or` operator parameter on the [`RefinmentList`](https://www.algolia.com/doc/api-reference/widgets/refinement-list/js/) widget.
+
+⚠️ This impacts the facet distribution number
+
+## Drawback
+
+Currently, for each `facet value` selected in a different `facet` a separate request is made to Meilisearch. Plus, an additional one containing all the facets filters.
```

Comment:
I would have added what it can be tedious to work with, especially regarding bandwidth cost

Reply:
```suggestion
Currently, for each `facet value` selected in a different `facet` a separate request is made to Meilisearch. Plus, an additional one containing all the facets filters. It results in tedious search requests and a high bandwidth cost.
```